tests/playbooks/k8s_auth.yml

```diff
@@ -15,17 +14,27 @@
     # It's good practice to store login credentials in a secure vault and not
     # directly in playbooks.
     #- include_vars: k8s_passwords.yml
-
+    - name: Get the hostname of master node dynamically
+      k8s_facts:
+        kind: Node
+        label_selectors:
+          - node-role.kubernetes.io/master = true
+      register: node_facts
+    - name: set fact for the hostname
+      set_fact:
+        hostname: "{{ 'https://' + node_facts.resources[0].status['addresses'][-1]['address'] + ':8443'}}"
     - name: Log in (obtain auth token)
       k8s_auth:
-        username: developer
+        host: "{{ hostname }}"
+        username: test_admin
         password: "{{ k8s_developer_password }}"
```

Comment:
We changed the use name so why are we referring to `k8s_developer_password`? Why do we hardcode user name?

Reply:
The purpose was to match the username in QE environment, the reason password was not changed was the user `test_admin` accepts any password.